vignettes/howto-dm-theory.Rmd

```diff
@@ -81,24 +83,24 @@ flights %>%
 
 {dm} offers a more elegant and shorter way to combine tables while augmenting {dplyr}/{dbplyr} workflows.
 
-It is possible to have the best of both worlds: manage your data with {dm} as linked tables, then flatten multiple tables into one for your analysis with {dplyr} on an as-needed basis.
+It is possible to have the best of both worlds: manage your data with {dm} as linked tables, and, when necessary, flatten multiple tables into a single data frame for analysis with {dplyr}.
 
 The next step is to create a [data model](#model) based on multiple tables:
 
 ## 2. Data Model {#model}
 
-A data model shows the structure between multiple tables that can be linked together.
+A data model shows the structure between multiple tables that are linked together.
 
 The `nycflights13` relations can be transferred into the following graphical representation:
 
-```{r warning=F, message=F}
+```{r warning=FALSE, message=FALSE}
 dm <- dm_nycflights13(cycle = TRUE)
 
 dm %>%
   dm_draw()
 ```
 
-The `flights` table is linked to three other tables: `airlines`, `planes` and `airports`.
+The `flights` table is linked to three other tables: `airlines`, `planes`, and `airports`. <!-- What about `weather` table? -->
```

Comment:
Good catch, this was before we had compound keys.

Reply:
Cool. I have added it to the list.